src/snowflake/cli/_plugins/nativeapp/sf_sql_facade.py

```diff
@@ -181,6 +186,129 @@ def get_account_event_table(self, role: str | None = None) -> str | None:
             return None
         return table
 
+    def create_schema(
+        self, name: str, role: str | None = None, database: str | None = None
+    ):
+        """
+        Creates a schema.
+        @param name: Name of the schema to create. Can be a fully qualified name or just the schema name, in which case the current database or the database passed in will be used.
+        @param [Optional] role: Role to switch to while running this script. Current role will be used if no role is passed in.
+        @param [Optional] database: Database to use while running this script.
```

Comment:
it's used as the default, but this seems to indicate that it's an override. We should align the description and the implementation (either rename this to default_database or use it as an override).

Reply:
updated